src/Lykke.Job.OperationsCache/Settings/JobSettings/OperationsCacheSettings.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
+using Lykke.SettingsReader.Attributes;
 
 namespace Lykke.Job.OperationsCache.Settings.JobSettings
 {
     public class OperationsCacheSettings
     {
+        [Optional]
```

Comment:
why settings optional?

Reply:
let's make all settings property as Required

Reply:
becouse after admins and another piople can not find this setting or the one who writes to the index HistoryCacheInstance

Reply:
ok